react/src/hooks/index.tsx

```diff
@@ -251,7 +260,49 @@ export const useBackendAIImageMetaData = () => {
         const { tags } = getImageMeta(imageName);
         return tags[1];
       },
+      getBaseImages: (tag: string, name: string) => {
```

Comment:
getBaseImages,getConstraints implemented a new function without modifying the getBaseImage,getCustomTag,getFilteredRequirementsTag function
This is because it's different enough that other pages will likely need to be modified as well.